POS/Integration/04_AssignUsers.md

```diff
@@ -1,65 +1,30 @@
 [!!Create a store in POS](./06_CreateStore.md)
+[!! Create user group](../../Core1Platform/AdministratingCore1/03_ManageGroups.md#create-user-group)
+[!! Add user to user group](../../Core1Platform/AdministratingCore1/03_ManageGroups.md#add-user-to-user-group)
+
 
-[comment]: <> (add link to Register a user, Settings module if available)
 
 # Assign users to the POS groups
 
-Users must belong to a specific POS group in order to be used as a cashier or store manager in the POS.
+Users must belong to a specific POS group in order to be registered as a cashier or store manager in the POS.
 The following POS groups are available:
 - POS Admin:  
-    The POS Admin has all rights in the POS and can configure the settings in all stores, assign store managers and cashiers.
+    The POS Admin has all rights in the POS and can configure the settings in all stores as well as assign store managers and cashiers.
 - POS Store Manager:   
-    The POS Store Manager can configure the store settings in the assigned store. He/She can assign cashiers, log out cashiers, create pay desks, view the shift summary, etc. He/She has no rights to change the admin settings in the store.
+    The POS Store Manager can configure the store settings in the assigned store. He/She can assign cashiers, log out cashiers, create pay desks, view the shift summary, etc. S/he has no rights to change the admin settings in the store.
 - POS Cashier:   
-    The POS cashier can log in and cash out on the assigned pay desk. He/She has no rights to change settings.
+    The POS cashier can log in and cash out on the assigned pay desk. S/he has no rights to change settings.
 
 The users that belong to a POS group must be assigned to a pay desk or store in order to have the above rights.
 
-> [Info] Note that you need system administrator rights to be allowed to make the assignment.  
-
-[comment]: <> (The following How-To Guide is not specific for POS and may be reused)
 
 #### Prerequisites
 
 - The users have been registered.
```

Comment:
Was bedeutet hier registered? Oben im Bezug auf POS (also, für POS registrieren/verwenden...).
Ist vielleicht "The user have been created" gemeint? 
Was ist der Unterschied zwischen "register as a cashier or manger in POS" und "assign users to a POS group, z.B. POS Admin, POS Store Manager, POS Cashier". Ich würde nur ein Verb verwenden, sonst scheinen unterschiedliche Sachen zu sein...
1. Create user (Settings)
2. Create group and assign user to a group (Settings)
3. Assign a user to a pay desk/store (POS -> in einem anderen Topic, oder?)